feature-libs/checkout/components/components/cost-center/cost-center.component.ts

```diff
@@ -31,9 +31,9 @@ export class CostCenterComponent {
       filter(([costCenters]) => Boolean(costCenters)),
       tap(([costCenters, cartCostCenter]) => {
         if (!Boolean(cartCostCenter)) {
-          this.setCostCenter(costCenters[0].code);
+          this.setCostCenter(costCenters[0].code as string);
         } else {
-          this.costCenterId = cartCostCenter;
+          this.costCenterId = cartCostCenter as string;
```

Comment:
Is it possible to add a type on line 32 instead of using a cast?

Reply:
even we add type, we still need to use cast, because `this.checkoutCostCenterService.getCostCenter()` return `string | undefined`. So, I change `costCenterId: string | undefined;`.